### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**
Screenshot 2022-10-26 210412
Screenshot 2022-10-26 210450
Screenshot 2022-10-26 210516
Screenshot 2022-10-26 210649
Screenshot 2022-10-26 210841
Screenshot 2022-10-26 210557

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+
+.sidebar-nav {
+    list-style: none;
+}
+
+.list_item {
+    list-style: none;
+}
+/*
+What do I feel here
```
Comment (mltdll) [inline static/css/styles.css:15]: Beautiful. Why didn't you include it in your website?

File: templates/taxi/car_detail.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                <p>
+                  <a href="{% url 'taxi:driver-detail' pk=driver.id %}">{{ driver.username }}</a><br>
+                  {{ driver.first_name }} {{ driver.last_name }}
+                </p>
+              </div>
+            </li>
+          {% endfor %}
+        </ul>
+  {% else %}
+    <p><strong>No one drive this car.</strong></p>
```
Comment (mltdll) [inline templates/taxi/car_detail.html:25]: You mean, no one drives?

File: templates/taxi/car_list.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+              {{ car.manufacturer.name }}<br>
+              <strong>Model: </strong>
+              <a href="{% url 'taxi:car-detail' pk=car.id %}">{{ car.model }}</a>
+            </p>
+          </div>
+        </li>
+      {% endfor %}
+    </ul>
+  {% else %}
+    <p>There no cars in list</p>
```
Comment (mltdll) [inline templates/taxi/car_list.html:21]: There **are** no cars. By the way, you forgot the period at the end of the sentence.

File: templates/taxi/driver_list.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="item-content">
+          <p>
+            <a href="{% url 'taxi:driver-detail' pk=driver.id %}">{{ driver.username }}</a><br>
+            {{ driver.first_name }} {{ driver.last_name }}
+          </p>
+        </div>
+      </li>
+    {% endfor %}
+    {% else %}
+      <div>It looks like nobody works here</div>
```
Comment (mltdll) [inline templates/taxi/driver_list.html:18]: Where did the `.` go?

File: templates/taxi/manufacturer_list.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+              <p><strong>Brand name:</strong>
+                {{ manufacturer.name }}<br>
+                <strong>Origin: </strong> {{ manufacturer.country }}
+              </p>
+            </div>
+        </li>
+        {% endfor %}
+    </ul>
+  {% else %}
+    <p>There are no manufacturers</p>
```
Comment (mltdll) [inline templates/taxi/manufacturer_list.html:19]: 50% of the messages about empty stuff in your website seem to end with period, 50% don't. Does that mean you're in superposition?

**Review by mltdll — CHANGES_REQUESTED** [review]
Looks good. At least the code part.

**Review by mltdll — APPROVED** [review]

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Comment by StasOkhrym** [comment]
Screenshot 2022-10-27 201945
It should be like this?

**Comment by Nattalli** [comment]
> Screenshot 2022-10-27 201945
> 
> It should be like this?

yes, kind of

**Review by Nattalli — APPROVED** [review]

**Pull request closed without merging**